test/workflow_definitions/workflow_definitions_test.exs

```diff
@@ -11,17 +11,29 @@ defmodule StepFlow.WorkflowDefinitionsTest do
   end
 
   describe "workflow_definitions" do
-    test "list_workflow_definitions/0 returns all workflow_definitions" do
+    test "list_workflow_definitions/0 returns workflow_definitions with rights" do
```

Comment:
create an another test.
This test is not anymore enough an unit test ...

Reply:
You mean there is both the process of listing workflows and using rights?